overlord/snapstate/check_snap.go

```diff
@@ -263,13 +263,14 @@ func checkCoreName(st *state.State, snapInfo, curInfo *snap.Info, flags Flags) e
 	// transition we will end up with not connected interface
 	// connections in the "core" snap. But the transition will
 	// kick in automatically quickly so an extra flag is overkill.
-	if snapInfo.Name() == "core" && core.Name() == "ubuntu-core" {
+	// TODO parallel-install: use store name
```

Comment:
core should probably not be installed in parallel at all, so using instance name is what we want